BugSplatDotNetStandard/Http/JsonObject.cs

```diff
@@ -29,5 +30,70 @@ public string GetValue(params string[] path)
             var key = string.Join("/", path);
             return root.XPathSelectElement($"//{key}").Value;
         }
+
+        public override string ToString()
+        {
+            return json;
+        }
+
+        public static JsonObject Create(Dictionary<string, string> dictionary)
+        {
+            return new JsonObject(JsonSerializer.Serialize(dictionary));
+        }
+    }
+
```

Comment:
Claude FTW!!! Adding new dependencies is a nightmare because of Unity. With an LLM that can write a JSON Serializer in < 1 minute we don't have to!